README.md

```diff
@@ -3,53 +3,67 @@
 
 ![Dark Reader screenshot](https://lh3.googleusercontent.com/tTJIIIAqfJWymqPM9LAmivl11kWmF-XXLABues4OwfjOED_ntsJZdLY0VTG0XFCW0W_wYSll7Q=w640-h400-e365)
 
-This extension **inverts brightness** of web pages and aims to **reduce eyestrain** while browsing the web.
+This extension **inverts brightness** of web pages and aims to **reduce eyestrain** while you browse the web.  
```

Comment:
Remove the spaces after `.`

Reply:
I added the spaces after several different lines so that in the GitHub web-app a newline is added.  AFAIK, that's the cleanest way to add a newline that is compatible with the GitHub web-app.

Reply:
Got it.